pkg/api/test/mock_admin_runner.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2017 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package test
+
+import "github.com/rook/rook/pkg/clusterd"
+
+type MockAdminCommandRunner struct {
```

Comment:
is this applicable to just RGW? or more general admin commands outside of the RGW subsystem?